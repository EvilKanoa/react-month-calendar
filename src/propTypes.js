import PropTypes from 'prop-types';

export default {
  /** Passes a custom ID to the rendered calendar. */
  id: PropTypes.string,
  /** Additional className to apply to the rendered calendar. */
  className: PropTypes.string,
  /** The number of days to be displayed in the rendered calendar. Controls the number of columns that you can render events into. */
  dayCount: PropTypes.oneOf([1, 2, 3, 4, 5, 6, 7, 8]),
};
