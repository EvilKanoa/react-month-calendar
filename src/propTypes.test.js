import propTypes from './propTypes';

describe('propTypes', () => {
  it('matches snapshot', () => {
    expect(propTypes).toMatchSnapshot();
  });
});
