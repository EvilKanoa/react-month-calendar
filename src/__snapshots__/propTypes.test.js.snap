// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`propTypes matches snapshot 1`] = `
Object {
  "className": [Function],
  "dayCount": [Function],
  "dayHeaderRenderer": [Function],
  "displayedInterval": [Function],
  "endMinutes": [Function],
  "id": [Function],
  "minutePrecision": [Function],
  "startMinutes": [Function],
  "translateDay": [Function],
  "weekStart": [Function],
}
`;
